OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -3538,91 +3545,244 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                </item>
+                                
+                                <item xml:id="e4">
+                                    <locus target="#8rb #9vb #14vb #31vb #32va #35va #48va #49vb #54vb #58ra #60ra #64vb #65ra #67ra #69va #71rab #71vab #72vb
+                                        #73ra #77va #78rb #80rab #90vb #93ra #93va #103rb #106va #108ra #109ra #110ra"/>
+                                    <!--always difficult to find the right moment to stop...-->
+                                    <desc>Scribal corrections occur regularly interlineally or in the margins.</desc> 
+                                </item>
+                                
+                                <item xml:id="e5">
+                                    <locus target="#58ra #70va #74vb"/>
+                                    <desc>Scribal corrections occur rarely over erasures.</desc> 
+                                </item>
+                                
+                                <item xml:id="e6">
+                                    <locus target="#45va #72rb #90vb #96rb #101arb #106ra"/>
+                                    <desc>Rarely, erroneous characters are marked by dashes above and below.</desc> 
+                                </item>
+                                
+                                <item xml:id="e7">
+                                    <locus target="#78ra"/>
+                                    <desc><foreign xml:lang="gez">ማርያም፡</foreign>
+                                        is added in a later hand in the space left for the rubricated name of Mary.</desc> 
+                                </item>
+                                
+                                <item xml:id="e8">
+                                    <locus target="#122v"/>
+                                    <desc>Note in pencil.</desc>
+                                    <q xml:lang="en">Miracles <!--I cannot read the next word--> <lb/> Blessed Virgin <lb/> XVI-XVII Cent. <lb/> 17 Pages
+                                        <lb/> <!--I cannot read the word or name--></q>
+                                </item>
+                                
+                                <item xml:id="e9">
+                                    <locus target="#1r"/>
+                                    <desc> A white sticker, with the name of <persName
+                                        ref="PRS5782JuelJen"/> (<foreign xml:lang="gez">ቤንት፡
+                                            ዩውል፡ የንሰን።</foreign>) printed on it, and the signature
+                                        ‘Ms. Aeth. d. 20 = Arch. O’ written in pencil, is  glued
+                                        on <locus target="#1r"/>. Another sticker with the signature ‘Ms 5’ printed on
+                                        it are glued below it.</desc>
```

Comment:
"is" glued below it
too many spaces within the description are not meaningful, right?

Reply:
If one space in the place is ok, too many spaces make no difference. Here for example, there was   `<desc> A white ` where the content started with a space, which is not great for consistency, I corrected it.